cfme/ansible_tower/explorer.py

```diff
@@ -58,6 +59,12 @@ class TowerExplorerSystemJobTemplatesToolbar(View):
     view_selector = View.nested(ItemsToolBarViewSelector)
 
 
+class TowerExplorerSystemJobTemplatesDetailsToolbar(View):
+    reload = Button(title='Refresh this page')
```

Comment:
Optional: `reload` was `build-in` function for py2 and moved to `importlib` in py3. still I would like to name such things with other name like `refresh` here. I know on same page its used multiple time. totally optional.